blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarter.java

```diff
@@ -295,8 +303,10 @@ public void run() {
                     TimeUnit.MINUTES.toMillis(savePeriodMins));
         }
 
-        // has the side-effect of causing static initialization of Metrics, starting instrumentation reporting.
-        new RestartGauge(Metrics.getRegistry(), RollupService.class);
+        if (shouldInstantiateRestartGauge) {
```

Comment:
If this flag is only for tests, I have an idea. We can create a separate method just for this one line "Metrics.getRegistry()" and then mock the return value from tests? You have to try this to see if it works.  Its worth a shot if it means getting rid of a flag.